explorer/Dockerfile

```diff
@@ -7,9 +7,7 @@ ENV PATH /usr/src/app/node_modules/.bin:$PATH
 ADD . .
 
 RUN yarn install
-RUN cd explorer \
-      && yarn install \
-      && yarn build
+RUN yarn workspace @chainlink/explorer run build
```

Comment:
Maybe we should remove the "prebuild" hook inside `explorer`? Seems like we run `yarn install` multiple times when we only need to once